tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/JSDesugaring.scala

```diff
@@ -2262,7 +2344,7 @@ private[emitter] class JSDesugaring(semantics: Semantics,
 
   private[emitter] def genRawJSClassConstructor(className: String,
       spec: Option[JSNativeLoadSpec])(
-      implicit pos: Position): js.Tree = {
+      implicit globalKnowledge: GlobalKnowledge, pos: Position): js.Tree = {
```

Comment:
This appears not to be used. Similarly, callers of `genRawJSClassConstructor` should be checked whether their new `implicit globalKnowledge` was added only to call `genRawJSClassConstructor`.